### DISCUSSION TITLE
A11y improvements for MarkerClusterer

### DISCUSSION THREAD

**Discussion by julliakharchenko** [JustFly1984/react-google-maps-api#2988] [Ideas]
`MarkerClusterer` is not reachable by the keyboard. 
User press `Tab` on the keyboard and can't reach the clusters on the map. This is happening because cluster div tag does not have a `tabindex` attribute. Also good to have `role="button"` with the tabindex for screen reader to read it properly. 
This will help improve a11y a lot! 
Pins has `tabindex` and a `role` and work really nice, but not the cluster.
![Screenshot 2022-04-12 at 19 20 10](https://user-images.githubusercontent.com/27816893/163019475-e9fba2dc-b2d0-461f-b497-93f976cd907c.png)

**Comment by JustFly1984** [comment]
Your PR is welcome
